src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -509,22 +510,23 @@ private async Task ProcessActionsAsync(LazyInput lazyInput, IContext context, Ac
                 {
                     try
                     {
-                        var settings = stateAction.Settings;
-                        if (settings != null)
+                        var jObjectSettings = new JObject();
```

Comment:
You don't need to allocate a new `JObject` here. I think this should work:
```csharp
JObject jObjectSettings = null;
```

Reply:
Done!